boot/bootdata/hivecls.inf

```diff
@@ -17,6 +17,7 @@ HKCR,"Folder\shell\open\command","",0x00000000,"explorer.exe ""%1"""
 HKCR,"Folder\shell\explore","BrowserFlags",0x00010001,"0x00000022"
 HKCR,"Folder\shell\explore","ExplorerFlags",0x00010001,"0x00000021"
 HKCR,"Folder\shell\explore\command","",0x00000000,"explorer.exe /e,""%1"""
+HKCR,"Folder\shell\Command Prompt Here...\command","",0x00000000,"cmd.exe /k ""cd /d ^""%1^"""""
```

Comment:
Three things:
* This should go under "Directory", not "Folder" since it is only valid for directories on the file system, not for shell-only folders
* Even though the ROS shell probably doesn't support this yet, there should be an empty string value called "Extended" that causes the menu item only to show up if Shift+Right Click is used
* The "Command Prompt Here..." string should be localizable. To achieve that, use a "shell32.dll,-X" string (as seen above for DefaultIcon) as the Default value under this key, and add a string to shell32.dll's resources.

Thanks.

Reply:
Does ROS need to hide it as an extended/shift key item, just because that's what Windows does? IMO there's no harm in always showing it.

Reply:
It would probably be a good idea to put it in so that when the functionality is added we'll have something to test it with.